pcdsdevices/mirror.py

```diff
@@ -1154,6 +1154,10 @@ class KBOMirrorHE(KBOMirror):
     cool_flow1 = Cpt(EpicsSignalRO, ':FWM:1_RBV', kind='normal')
     cool_press = Cpt(EpicsSignalRO, ':PRSM:1_RBV', kind='normal')
 
+    #
```

Comment:
nitpick
```suggestion
```

Reply:
Where'd this little dude come from. Updated